docs/book/src/reference/testing/integration.md

```diff
@@ -77,4 +77,62 @@ func TestReconcile(t *testing.T) {
 
 The manager is started as part of the test itself (`StartTestManager` function).
 
-Both functions are located in `pkg/controller/parent/parent_controller_suite_test.go` file. The file also contains a `TestMain` function that allows you to specify CRD directory paths for the testing environment.
\ No newline at end of file
+Both functions are located in `pkg/controller/parent/parent_controller_suite_test.go` file. The file also contains a `TestMain` function that allows you to specify CRD directory paths for the testing environment.
+
+## Using envtest in your integration tests
+[`controller-runtime`]() offers `envtest` ([godoc](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg/envtest)), a package that helps write integration tests for your controllers by setting up and starting an api-server binary and an etcd binary.
```

Comment:
nit: "an instance of etcd and the Kubernetes API server, without kubelet, controller-manager etc" reads better IMO.

Reply:
Thanks! I rephrased it.